### DISCUSSION TITLE
Is it possible to make Hyper appear when typing CMD into the File Explorer, rather than the pre-set CMD?

### DISCUSSION THREAD

**Discussion by HalfTurk** [vercel/hyper#5808] [Q&A]
Hello,

I've been using Hyper for a couple of days now and I was wondering if it was somehow possible to make the Hyper open when you type CMD into the File Explorer directory area. If anyone knows how to accomplish this, please let me know :).

Edit: Link: https://user-images.githubusercontent.com/61254714/127753557-4a2dd92c-1777-40f6-9c50-6982956f1c35.gif

Thanks, Kap

**Comment by LabhanshAgrawal** [comment]
Not sure, but you can try `hyper - v`
Anyways you can right click and use 'open hyper here' from menu

**Reply by HalfTurk** [reply]
I know but I still just want to make it the default thing you know? Like perhaps there could be a way to change the CMD without fully deleting it? Or just change like a secret environment variable, which I couldn't find anywhere, to hyper.exe?

**Comment by asdf23** [comment]
Windows has a debug feature in the registry.. Instead of running abc.exe run zyx.exe. I bet that would work.

**Reply by asdf23** [reply]
@KappasDEV https://hmemcpy.com/2010/12/how-to-debug-a-process-that-is-crashing-on-startup/ I bet you could get this to work. The idea is instead of cmd.exe run hyper (the instructions are aimed at a different goal but at least you'll get the idea.) Let me know if you get this working!

**Reply by HalfTurk** [reply]
Uh I’m just asking how I could open Hyper by default instead of CMD… I don’t get what you mean…
